deps/graph-parser/src/logseq/graph_parser/property.cljs

```diff
@@ -62,6 +62,48 @@
      :todo :doing :now :later :done}
    @built-in-extended-properties))
 
+(def built-in-property-types
```

Comment:
Listing all the built-in properties that are parsed is a smaller list than all the properties that should not be parsed. Added types since it is good documentation and could be helpful if we look to type properties